src/OpenTelemetry.Instrumentation.GrpcNetClient/OpenTelemetry.Instrumentation.GrpcNetClient.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
It looks like the grpc.net library added netstandard2.0 target recently. 
We need to make sure tests are run against .NET FW as well.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/test/OpenTelemetry.Instrumentation.Grpc.Tests/OpenTelemetry.Instrumentation.Grpc.Tests.csproj#L4

Reply:
Looking into it - not super trivial because the server part doesn't work on .NET Framework and the tests depends on a server running...

Reply:
I can't test if the tests work now as my windows version is too old: https://docs.microsoft.com/en-gb/aspnet/core/grpc/netstandard?view=aspnetcore-5.0

I didn't actually realise that there are also two different clients for gRPC (just like for the server - native and managed). I have always used the native gRPC client under .net framework. Is it possible to instrument that?

Reply:
> It looks like the grpc.net library added netstandard2.0 target recently.

Very cool!

> I have always used the native gRPC client under .net framework. Is it possible to instrument that?

The contrib project has instrumentation for the other gRPC client. See: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Contrib.Instrumentation.GrpcCore